### PR TITLE
Add intellij section with folder plugin

### PR DESCRIPTION
**Pull request by heldev**
Would be great to have a section dedicated to Java specific plugins.

Let’s start with this one. It gives a new more concise appearance to your existing Java codebase.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Frameworks and libraries that simplify reading/writing CSV data.*
 
+* [jackson-dataformat-csv](https://github.com/FasterXML/jackson-dataformat-csv) - Jackson extension for reading and writing CSV (either as a sequence of Strings, or via data binding to/from POJOs).
```
Comment (akullpp) [inline README.md]: Remove this sideload from the PR since adding a new section is quite much.

File: README.md [hunk 2]
```diff
@@ -431,6 +432,10 @@ A curated list of awesome Java frameworks, libraries and software.
 * [IntelliJ IDEA ![c]](http://www.jetbrains.com/idea/) - Supports a lot of JVM languages and provides good options for Android development. The commercial edition targets the enterprise sector.
 * [NetBeans](https://netbeans.org/) - Provides integration for several Java SE and EE features from database access to HTML5.
 
+## IntelliJ IDEA Java Plugins
```
Comment (akullpp) [inline README.md]: Link in content.
Description.
2 more plugins - not just any plugins. Viable and used ones please.

**Comment by koppor** [comment]
Please try to add a link to/from https://github.com/maksimr/awesome-intellij somehow.

**Comment by akullpp** [comment]
Glad that there's this list, so we don't need the section at all then.

**Pull request closed without merging**